purchase_order_reorder_lines/__openerp__.py

```diff
@@ -0,0 +1,40 @@
+# -*- coding: utf-8 -*-
+#
+#
+#    Author: Alexandre Fayolle
+#    Copyright 2013 Camptocamp SA
+#
+#    Author: Damien Crier
+#    Copyright 2015 Camptocamp SA
+#
+#    This program is free software: you can redistribute it and/or modify
+#    it under the terms of the GNU Affero General Public License as
+#    published by the Free Software Foundation, either version 3 of the
+#    License, or (at your option) any later version.
+#
+#    This program is distributed in the hope that it will be useful,
+#    but WITHOUT ANY WARRANTY; without even the implied warranty of
+#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#    GNU Affero General Public License for more details.
+#
+#    You should have received a copy of the GNU Affero General Public License
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+#
+{
+    'name': 'Purchase order lines with sequence number',
+    'version': '0.1',
+    'category': 'Purchase Management',
+    'author': "Camptocamp,Odoo Community Association (OCA)",
+    'website': 'http://www.camptocamp.com',
+    'depends': [
+        'purchase',
+        'stock_picking_reorder_lines',
```

Comment:
Hello,
Original module in v7 already had this dependency.
It depends on  'stock_picking_reorder_lines' because that module add 'sequence' field in "stock.move" object that is used in this module when confirming a purchase.
Without this dependency, you cannot be sure there is a 'sequence' field in 'stock.move' object and Odoo server always raise a warning in the log saying "write to an unexisting field 'sequence' on object 'stock.move'"